docs/Rules/ProvideCommentHelp.md

```diff
@@ -17,9 +17,9 @@ presence of comment based help and not on the validity or format.
 For assistance on comment based help, use the command `Get-Help about_comment_based_help` or the
 following articles:
 
-- [Writing Comment-based Help](https://docs.microsoft.com/powershell/scripting/developer/help/writing-comment-based-help-topics)
-- [Writing Help for PowerShell Cmdlets](https://docs.microsoft.com/powershell/scripting/developer/help/writing-help-for-windows-powershell-cmdlets)
-- [Create XML-based help using PlatyPS](https://docs.microsoft.com/powershell/scripting/dev-cross-plat/create-help-using-platyps)
+- [Writing Comment-based Help](https://learn.microsoft.com/powershell/scripting/developer/help/writing-comment-based-help-topics)
+- [Writing Help for PowerShell Cmdlets](https://learn.microsoft.com/powershell/scripting/developer/help/writing-help-for-windows-powershell-cmdlets)
+- [Create XML-based help using PlatyPS](https://learn.microsoft.com/powershell/utility-modules/platyps/create-help-using-platyps)
```

Comment:
This change was NOT made in Microsoft docs: https://github.com/MicrosoftDocs/PowerShell-Docs-Modules/blob/main/reference/docs-conceptual/PSScriptAnalyzer/Rules/ProvideCommentHelp.md?plain=1#L22
```suggestion
- [Create XML-based help using PlatyPS](https://learn.microsoft.com/powershell/utility-modules/platyps/create-help-using-platyps)
```

Reply:
The current URL redirects to the correct location. The new URL will come over when 1.21 is released. So the change to docs can wait for the release.

Reply:
release of platyps? This is getting too complicated! Can we not just have a clean PR in PSSA please that replicates what your other linked PR does, which is just changing the URL to learn.microsoft.com? And anything else that is still needed but has a different timeline in a separate PR?

Reply:
As far as I can tell this PR is clean. Sorry, you misunderstood which release I am talking about. I was talking about the 1.21 of PSSA. The link to the PlatyPS article in the docs repo will redirect so it doesn't need to change right now. But I can change it if you feel strongly. My point is that the change in this PR will get copied to the docs repo when PSSA 1.21 releases.

Reply:
ok, gotcha, that means we have to copy this file and the shouldprocess doc: https://github.com/MicrosoftDocs/PowerShell-Docs-Modules/pull/99